packs/data/the-enmity-cycle-bestiary.db/sand-wolf.json

```diff
@@ -0,0 +1,583 @@
+                            "acid",
+                            "electricity",
+                            "piercing",
+                            "slashing"
+                        ],
+                        "value": 8
+                    }
+                ],
+                "slug": null,
+                "source": {
+                    "value": ""
+                },
+                "traits": {
+                    "value": []
+                },
+                "trigger": {
+                    "value": ""
+                }
+            },
+            "type": "action"
+        },
+        {
+            "_id": "CSxBQgkOH3OC3Wpm",
+            "img": "systems/pf2e/icons/default-icons/action.svg",
+            "name": "Water Vulnerability",
+            "sort": 600000,
+            "system": {
+                "actionType": {
+                    "value": "passive"
+                },
+                "actions": {
+                    "value": null
+                },
+                "category": "defensive",
+                "description": {
+                    "value": "<p>Sand wolves take 5 additional damage from attacks and effects with the water trait.</p>"
+                },
+                "requirements": {
+                    "value": ""
+                },
+                "rules": [
+                    {
+                        "key": "Weakness",
+                        "type": "water",
+                        "value": 5
+                    }
+                ],
+                "slug": null,
+                "source": {
+                    "value": ""
+                },
+                "traits": {
+                    "value": []
+                },
+                "trigger": {
+                    "value": ""
+                }
+            },
+            "type": "action"
+        },
+        {
+            "_id": "Qwa6uCGVwCVxgKZg",
+            "img": "systems/pf2e/icons/default-icons/action.svg",
+            "name": "Choking Sands",
+            "sort": 700000,
+            "system": {
+                "actionType": {
+                    "value": "action"
+                },
+                "actions": {
+                    "value": 2
+                },
+                "category": "offensive",
+                "description": {
+                    "value": "<p>The sand wolf discorporates into a cloud of whirling sand. It can Stride, moving freely through occupied spaces. Any creatures within those spaces take [[/r 2d10]] damage and must attempt a @Check[type:reflex|dc:19] save.</p>\n<p><strong>Critical Success</strong> The target takes no damage.</p>\n<p><strong>Success</strong> The target takes half damage.</p>\n<p><strong>Failure</strong> The target takes full damage and is @UUID[Compendium.pf2e.conditionitems.Sickened]{Sickened 1}.</p>\n<p><strong>Critical Failure</strong> As failure, but the effect is increased to @UUID[Compendium.pf2e.conditionitems.Sickened]{Sickened 3}.</p>"
```

Comment:
```suggestion
                    "value": "<p>The sand wolf discorporates into a cloud of whirling sand. It can Stride, moving freely through occupied spaces. Any creatures within those spaces take [[/r 2d10]] damage and must attempt a @Check[type:reflex|dc:19] save.</p>\n<hr>\n<p><strong>Critical Success</strong> The target takes no damage.</p>\n<p><strong>Success</strong> The target takes half damage.</p>\n<p><strong>Failure</strong> The target takes full damage and is @UUID[Compendium.pf2e.conditionitems.Sickened]{Sickened 1}.</p>\n<p><strong>Critical Failure</strong> As failure, but the effect is increased to @UUID[Compendium.pf2e.conditionitems.Sickened]{Sickened 3}.</p>"
```